ush/templates/anavinfo.fv3lam_hrrr

```diff
@@ -2,56 +2,56 @@ met_guess::
 !var     level    crtm_use    desc              orig_name
   ps        1      -1         surface_pressure     ps
   z         1      -1         geopotential_height  phis
-  u        64       2         zonal_wind           u
-  v        64       2         meridional_wind      v
-  tv       64       2         virtual_temperature  tv
-  q        64       2         specific_humidity    sphu
-  oz       64       2         ozone                ozone
-  delp     64      -1         fv3 del pressure     delp
+  u        65       2         zonal_wind           u
+  v        65       2         meridional_wind      v
+  tv       65       2         virtual_temperature  tv
+  q        65       2         specific_humidity    sphu
+  oz       65       2         ozone                ozone
+  delp     65      -1         fv3 del pressure     delp
 ::
 
 state_derivatives::
 !var  level  src
  ps   1      met_guess
- u    64     met_guess
- v    64     met_guess
- tv   64     met_guess
- q    64     met_guess
- oz   64     met_guess
- cw   64     met_guess
- prse 65     met_guess
+ u    65     met_guess
+ v    65     met_guess
+ tv   65     met_guess
+ q    65     met_guess
+ oz   65     met_guess
+ cw   65     met_guess
+ prse 66     met_guess
 ::
 
 state_tendencies::
 !var  levels  source
- u    64      met_guess
- v    64      met_guess
- tv   64      met_guess
- q    64      met_guess
- oz   64      met_guess
- prse 65      met_guess
+ u    65      met_guess
+ v    65      met_guess
+ tv   65      met_guess
+ q    65      met_guess
+ oz   65      met_guess
+ prse 66      met_guess
 ::
 
 state_vector::  
 !var     level  itracer source     funcof
- u        64      0     met_guess    u
- v        64      0     met_guess    v
- tv       64      0     met_guess    tv
- tsen     64      0     met_guess    tv,q
- q        64      1     met_guess    q
- oz       64      1     met_guess    oz
- prse     65      0     met_guess    prse
+ u        65      0     met_guess    u
+ v        65      0     met_guess    v
+ tv       65      0     met_guess    tv
+ tsen     65      0     met_guess    tv,q
+ q        65      1     met_guess    q
+ oz       65      1     met_guess    oz
+ prse     66      0     met_guess    prse
  ps        1      0     met_guess    prse
  sst       1      0     met_guess    sst
 ::
 
 control_vector::
 !var     level  itracer as/tsfc_sdv  an_amp0   source  funcof
- sf       64      0       1.00        -1.0     state    u,v
- vp       64      0       1.00        -1.0     state    u,v
+ sf       65      0       1.00        -1.0     state    u,v
+ vp       65      0       1.00        -1.0     state    u,v
  ps        1      0       0.50        -1.0     state    prse
- t        64      0       0.70        -1.0     state    tv
- q        64      1       0.70        -1.0     state    q
+ t        65      0       0.70        -1.0     state    tv
+ q        65      1       0.70        -1.0     state    q
```

Comment:
Is this the desired outcome for all applications using surface cycling? Is there any possibility to use this branch with surface cycling turned off if these values are hardcoded?

Reply:
This is for changing to 65 levels in model forecast.

Reply:
Gotcha. Not related to the surface cycling. But are all GSI applications going to need to be doing this?